UPGRADE-3.x.md

```diff
@@ -18,6 +22,34 @@ This command was introduced a long time ago to migrate from array to json. Make
 
 This class is deprecated because it is dead code. If you use it, please use `ProviderDataTransformer` instead.
 
+### Deprecated using `master` and `slave` words in configuration
+
+Before:
+
+```yaml
+sonata_media:
+    cdn:
+        fallback:
+            master: sonata.media.cdn.panther
+    filesystem:
+        replicate:
+            master: sonata.media.adapter.filesystem.s3
+            slave: sonata.media.adapter.filesystem.local
+```
+
+After:
+
+```yaml
+sonata_media:
+    cdn:
+        fallback:
+            primary: sonata.media.cdn.panther
```

Comment:
IMO, we should not use this CDN in the "After" example, as it is also deprecated.

Reply:
This has to be solved on 3.x branch as this is just a manual merge.